src/advanced_agent/optimization/batch_processor.py

```
@@ -0,0 +1,669 @@
+            chunk_results = await asyncio.gather(*tasks, return_exceptions=True)
+            
+            # 結果を統合
+            results = []
+            for chunk_result in chunk_results:
+                if isinstance(chunk_result, Exception):
+                    logger.error(f"並列処理エラー: {chunk_result}")
+                    results.extend([None] * len(chunk_result))
+                else:
+                    results.extend(chunk_result)
+            
+            return results
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)